Mathlib/MeasureTheory/Constructions/Prod/Basic.lean

```diff
@@ -509,6 +509,19 @@ lemma _root_.MeasureTheory.NullMeasurableSet.right_of_prod {s : Set α} {t : Set
   refine ⟨Prod.mk x ⁻¹' u, measurable_prod_mk_left hum, ?_⟩
   rwa [mk_preimage_prod_right hxs] at hx
 
+/-- If `Prod.snd ⁻¹' t` is a null measurable set and `μ ≠ 0`, then `t` is a null measurable set. -/
+lemma _root_.MeasureTheory.NullMeasurableSet.of_preimage_snd [NeZero μ] {t : Set β}
```

Comment:
for the sake of symmetry, can you write down the same lemma for `fst` (even if it is only through an alias)?